docs/pages/contributing/documentation/style-guide.mdx

```diff
@@ -5,108 +5,152 @@ description: Use this reference to ensure our docs are consistent and effective
-  same content in the current paragraph or section.
-
-  You can include a link in the body text of a how-to guide as long as you tell
-  the reader why to follow the link and what information to glean from it.
-  "Follow the installation instructions in the AWS documentation" does this,
-  while "Read the AWS documentation for more information" does not.
-
-- Example code and configurations should be complete and pastable.
-
-- Each step is clearly stated so users can easily follow the sequence. Section
-  headings should have a format similar to "Step 1/3. Add a local user"
-
-#### Example outline
-
-- Guide title: "Access Kubernetes on GKE"
-- The purpose of the guide
-- Prerequisites
-- Setup steps
-- Next steps
-
-### Getting started articles
-
-**Getting started guides** are designed to get a user up and running in the
-shortest way possible. They are intended for first-time Teleport users, and are
-often one of the first interactions they will have with the product. They should
-be fun and stress free.
-
-<Admonition type="note">
-  "Getting started" is a better term than "Quickstart".
-  Google "Getting started with React" vs. "React quickstart"
-  and compare the search results.
-</Admonition>
-
-#### Expectations
-
-Getting started articles have the same expectations as how-to guides, plus the
-following:
-
-- It should be possible to complete the tutorial in 5–10 minutes.
-- Consider including a demo video in order to reach readers who prefer video.
-
-#### Example outline
-
-- Title: "SSO and Audit for Kubernetes in three steps"
-- Demo video
-- Prerequisites section: lists the tools used in the guide
-- Several sub-sections describing concrete steps that a user can verify they have completed
-- Wrap up with "Next steps" section.
-
-### Architecture guides
-
-**Architecture guides** explain how Teleport works to advanced readers, e.g.,
-architects, DevSecOps engineers, and SREs. They describe high-level concepts
-with a narrow focus on architectural concerns, and are used to evaluate
-architectural decisions, configuration, and usage scenarios.
-
-Specific kinds of architecture guides include:
-
-- **Networking**: Readers are interested in learning about specific ports, components, and protocols that are or can be used.
-- **Security**: Lists security protocols and primitives. SecOps will look for an attack vector tree diagram.
-- **Deployment**: Should include a deployment diagram which in turn explains components and how they interact with databases and each other.
-
-### Conceptual guides
-
-**Conceptual guides** are a continuations of our architectural guides and distinct from how-to guides. They explain a concept that is relevant for understanding Teleport. Some overlap will be likely between conceptual guides and architecture guides.
+## Information types and pages
+
+Depending on your point of view and goals, information can be classified in different ways.
```

Comment:
I think we can remove this sentence since it sounds pretty self-evident.

I would add a more explicit intro paragraph here that tells a reader what to expect in the H3s within this H2.